ExportScripts/export_standalone_monolithic_windows.py

```diff
@@ -27,7 +27,7 @@
 # View the argparse parameters for options available. An example invocation:
 
 # @<O3DE_ENGINE_ROOT_PATH>
-# > python\python.cmd <O3DE_MPS_PROJECT_ROOT_PATH>\ExportScripts\export_standalone_monolithic_windows.py -ps <O3DE_MPS_PROJECT_ROOT_PATH> -egn <O3DE_ENGINE_ROOT_PATH> -bnmt -out <MPS_OUTPUT_RELEASE_DIR_PATH> -zip
+# > python\python.cmd <O3DE_MPS_PROJECT_ROOT_PATH>\ExportScripts\export_standalone_monolithic_windows.py -pp <O3DE_MPS_PROJECT_ROOT_PATH> -ep <O3DE_ENGINE_ROOT_PATH> -bnmt -out <MPS_OUTPUT_RELEASE_DIR_PATH> -a -aof zip
```

Comment:
it seems like we could just have a single parameter e.g. `--archive-output-format`, and if it isn't set then we don't make an archive rather than requiring 2 parameters - unless we change how `archive-output-format` works to where it defaults to something

Reply:
It looks like if you don't provide a default value for the archive format the script will error out:
```
>>> shutil.make_archive('f:/Amazon/test', None, 'f:/Amazon/Lumberyard_1.27')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python38\lib\shutil.py", line 1034, in make_archive
    raise ValueError("unknown archive format '%s'" % format) from None
ValueError: unknown archive format 'None'
```

Reply:
updating it to set the default to zip if not specified.